src/core/model.jl

```diff
@@ -30,6 +30,14 @@ function correct_pos_type(n, model)
     end
 end
 
+function pos_vertex(n, model)
```

Comment:
I am confused, what does this function aim to do that `coord2vertex` can't?

Reply:
With this, you don't have to check whether `pos` is a tuple, and then convert it.

Reply:
https://github.com/JuliaDynamics/Agents.jl/blob/master/src/core/space.jl#L256 ? `coord2vertex` does this anyway.

Reply:
Oops. I missed that. I'll remove this function then.

Reply:
`coord2vertex` has problems when given an integer:

```jl
using Agents
mutable struct Ag <: AbstractAgent
  id
  pos
end
m = ABM(Ag, Space((2,3)))
coord2vertex(2, m)
ERROR: MethodError: coord2vertex(::Int64, ::AgentBasedModel{Ag,Agents.GridSpace{LightGraphs.SimpleGraphs.SimpleGraph{Int64},2,Int64},typeof(fastest),Nothing}) is ambiguous. Candidates:
  coord2vertex(c, model::AgentBasedModel) in Agents at D:\projects\Agents.jl\src\core\space.jl:219
  coord2vertex(coord::Integer, args...) in Agents at D:\projects\Agents.jl\src\core\space.jl:256
  coord2vertex(c, model::AgentBasedModel) in Agents at D:\projects\Agents.jl\src\core\space.jl:219
Possible fix, define
  coord2vertex(::Integer, ::AgentBasedModel)
Stacktrace:
 [1] top-level scope at REPL[10]:1
```

Reply:
Ops yea, its exactly because of 2 conflicting abstractions, but as the guide says, just define `coord2vertex(n::Integer, ::ABM) = n` and it will solve it.